vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.js

```diff
@@ -23,6 +23,14 @@ import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js
     static get properties() {
       return {
 
+        /**
+         * Automatically sets the width of the column based on the column contents when this is set to `true`.
+         */
+        autoWidth: {
+          type: Boolean,
+          value: true
+        },
+
```

Comment:
The change seems to work fine 👍  One more fine-tune: Once the initial data is received, it looks like the selection column snaps from the default 58px to 56px (by auto-width). So let's align the default width to be `56px` also to avoid the jump in most cases.

Reply:
done with the newest commit.

Reply:
done with the newest commit.